services/qs-spark-service/src/main/java/ai/quantumics/api/service/impl/AwsConnectionServiceImpl.java

```diff
@@ -45,43 +44,40 @@ public AwsDatasourceResponse saveConnectionInfo(AwsDatasourceRequest awsDatasour
     }
 
     @Override
-    public AwsDatasourceResponse updateConnectionInfo(AwsDatasourceRequest awsDatasourceRequest, Integer id, Projects project) throws ConnectionNotFoundException {
+    public AwsDatasourceResponse updateConnectionInfo(AwsDatasourceRequest awsDatasourceRequest, Integer id, String userName) throws ConnectionNotFoundException {
+
+        AWSDatasource dataSource = awsConnectionRepo.findByIdAndActive(id,true).orElseThrow(() -> new ConnectionNotFoundException("Connection not found"));
```

Comment:
"Connection not found" - invalid statement

Reply:
Message updated.